.github/workflows/lint-client.yml

```diff
@@ -42,4 +42,6 @@ jobs:
         run: yarn prettier:ci
 
       - name: Run Typechecks
-        run: yarn typecheck
+        run: |
+          yarn workspace graph-scaffolder tsc --build
```

Comment:
@dgauldie issue is that the types need to be built otherwise it does not properly reference them... not sure what the best option for this is really as the above is not really maintainable

Reply:
moved it to the script in package.json makes more sense there